src/templates/block-comment.jsx

```diff
@@ -1,25 +1,17 @@
 import React from 'react';
+import classNames from 'classnames';
 
 import { html } from '../indices-to-html';
 import { p } from './functions';
 
-export const CommentBlock = React.createClass({
-  render: function() {
-    var commentText = p(html(this.props.block));
-
-    var className = 'wpnc__comment';
-    if (this.props.meta.ids.comment != this.props.block.meta.ids.comment) {
-      className += ' comment-other';
-    } else {
-      className += ' comment-self';
-    }
-
-    return (
-      <div className={className}>
-        {commentText}
-      </div>
-    );
-  },
-});
+export const CommentBlock = ({ block, meta }) =>
```

Comment:
this is how I write my modules. I make sure that every export is a named export, then I often provide a default export. this is partially to support a pattern and partially to support testing